### PR TITLE
Minor fixes for workflow

### PR DESCRIPTION
**Pull request by wtrocki**
Currently error fetching will happen instantly after previous PR is merged. 
This is not ideal as errors in the API would not be online by then and we will get errors updated in the next run. I think this is just temporary tradeoff that will also reduce another redundant PR

### CODE REVIEW

File: .github/workflows/api-update.yaml [hunk 1]
```diff
@@ -19,6 +19,10 @@ jobs:
         run: ./scripts/fetch_api.sh ${{ github.event.client_payload.download_url }}
       - name: Generate API clients
         run: yarn generate
+      - name: Fetch Errors JSON files
```
[the remark this answers is not in the document]
Comment (dimakis) [reply .github/workflows/api-update.yaml:22]: https://stackoverflow.com/questions/69490965/can-we-wait-for-another-repo-workflow-to-complete-before-trigger-the-workflow-in 

it appears that people have had this issue before

[the remark this answers is not in the document]
Comment (wtrocki) [reply .github/workflows/api-update.yaml:22]: Deployment can happen in 1 hour timeframe. Ideally we want to have errors represented as another file in the original repo. So we really need to get that sorted in every source repository. We use url option right now - mixing file approach and url approach is problematic.

**Comment by wtrocki** [comment]
With this approach we can have errors changes landing with different SDK update (next subsequent update can regenerate it. options for it would be:

- run job once a week (too long leads)
- add errors.json file to the source repos and include it in the change event
- ??

File: .github/workflows/api-update.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (wtrocki) [reply .github/workflows/api-update.yaml:22]: Proposed wait is good for 2-3 minutes but not for waiting long periods. We do not know how long to way as we are waiting for gitlab/jenkins deployment

**Pull request merged**